src/functions/accent.js

```diff
@@ -63,9 +63,11 @@ const htmlBuilder = (group, options) => {
     }
 
     // calculate the amount of space between the body and the accent
-    const clearance = Math.min(
-        body.height,
-        options.fontMetrics().xHeight);
+    let clearance = group.value.label === "\\c"
+        ? body.height + body.depth
+        : Math.min(
+            body.height,
+            options.fontMetrics().xHeight);
```

Comment:
Is there a reason that this needs to be special-cased for `\c`?  Can we come up with a general rule?  Ah, I see, there's a fundamental difference according to whether the accent is above or below...  Maybe there should be an `const accentBelow = (group.value.label === "\\c")` which we'll later add more cases to?  Alternatively, `const accentBelow = (accent.depth > 0)` might work...

Reply:
Good call.